### PR TITLE
Gets events from Meetup API. Removes JSON event data

### PR DESCRIPTION
**Pull request by antonjb**
When starting the site you now have to include a Meetup API key. See
https://secure.meetup.com/meetup_api/key/. This will need to be added to
Netlify's build process.

There didn't appear to be webhooks available on Meetup so a process for
building will need to be looked at. Potentially Zapier.

### CODE REVIEW

File: src/components/EventCard.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 
-const EventCard = ({ active, month, url, type, title, date }) => {
+const EventCard = ({ active, month, url, title, date }) => {
+    const icons = ['social', 'workshop', 'talk']
+    const iconClass = icons[Math.floor(Math.random() * icons.length)]
```
Comment (antonjb) [inline src/components/EventCard.js]: There's no way to tell the type of event so I'm just randomly showing an icon. Suggestions?

File: plugins/gatsby-source-meetup/gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const crypto = require('crypto')
```
Comment (antonjb) [inline plugins/gatsby-source-meetup/gatsby-node.js]: Couldn't figure out the other Gatsby API plugins so now we have our own. Yay?

File: README.md [hunk 3]
```diff
@@ -2,13 +2,10 @@
...
 
 ## TODO
-- [x] Setup components
 - [ ] Convert CSS to Emotion or Styled Components
-- [ ] Pull Meetup events in from Meetup.com
 
 ## Running in development
-`gatsby develop`
-
+`MEETUP_API={API KEY} gatsby develop`
```
Comment (antonjb) [inline README.md]: Important

File: src/components/EventCard.js (hunk 1 above)
Comment (benderham) [reply src/components/EventCard.js]: At the moment the only 'social' event we are running is 'Casual Catchup'. I suggest making default the icon to be 'talk', then switch to social if title is 'Casual Catchup' and workshop if the the title includes 'Workshop'.

File: README.md (hunk 3 above)
Comment (benderham) [reply README.md]: Im pretty sure we can add the 'MEETUP_API' as an environment var in Netlify. Will have to get @amykapernick onto that one, or I'll get the login details.

File: plugins/gatsby-source-meetup/gatsby-node.js (hunk 2 above)
Comment (benderham) [reply plugins/gatsby-source-meetup/gatsby-node.js]: Sure, why not? If it does the job 👍

File: src/components/EventCard.js (hunk 1 above)
Comment (antonjb) [reply src/components/EventCard.js]: Sure, I can do that. Shouldn't be too tricky.

**Pull request merged**